### DISCUSSION TITLE
cross-repository search does not seem to work

### DISCUSSION THREAD

**Discussion by stapelberg** [community/community#9867] [Other Feature Feedback, Questions, & Ideas] [answered]
1. I open cs.github.com and type robustirc.
1. The first hit is https://cs.github.com/robustirc/robustirc/blob/4c327116eb52b7504b9659cbe75cf5781fe8b4de/robustirc.go#L4
1. I click e.g. `flag.String`, but I don’t see the Go standard library’s `flag.String` declaration in the definition list.

I thought that Go code was supported for cross-repository navigation? Or are there other requirements aside from the language?

**Comment by colinwm** [comment, marked as answer]
Cross-repo code navigation is only supported in python right now... but we're working on expanding it to more languages! Sorry about that.

**Comment by hughetop** [comment]
By "cross-repo code navigation" do you mean like a python import linking to the appropriate python file? Because I've never seen that work properly, it just references itself. To be honest I'd rather just disable the feature entirely for python imports since they don't work correctly.
